nvtabular/ops/categorify.py

```diff
@@ -38,10 +37,11 @@
 from merlin.core import dispatch
 from merlin.core.dispatch import DataFrameType, annotate, is_cpu_object, nullable_series
 from merlin.core.utils import device_mem_size, run_on_worker
+from merlin.dag.stat_operator import StatOperator
```

Comment:
`StatOperator` moved to Merlin Core as part of the base ops that are available to all Merlin DAGs (including NVT, Systems, etc)